src/assets/scss/application.scss

```diff
@@ -239,6 +239,29 @@ $nav-active-border-thickness: 5px;
   }
 }
 
+.account-header-navigation__button {
+  @include govuk-font($size: 19, $weight: bold);
+  border: 0 none;
+  cursor: pointer;
+  background-color: unset;
+}
+
+.account-header-navigation__button {
+  @include govuk-font($size: 16, $weight: bold);
+  border: 0 none;
+  cursor: pointer;
+  color: govuk-colour(white);
+  padding: 0;
+}
+
+.account-navigation__button {
```

Comment:
Some of the button reset styling is repeated between `.account-header-navigation__button` and `.account-navigation__button` so it could be possible to do
```
.account-header-navigation__button,
.account-navigation__button {
   ...stuff...
}
```

Reply:
Done